src/core_modules/capture-core/events/eventRequests.js

```diff
@@ -146,9 +146,10 @@ export async function getEvents(queryParams: ?Object) {
             accEvents.push(eventContainer);
         }
         return accEvents;
-    }, Promise.resolve([])) : null;
+    }, Promise.resolve([])) : [];
```

Comment:
This ties to my comments about trying to have keep the value type and not assign with `null`. Dont know where you stand on this but empty array for me is more meaningful than null because the annotation gives me more information about what is the type.